lib/Zonemaster/Backend/Config.pm

```diff
@@ -11,6 +11,13 @@ use File::ShareDir qw[dist_file];
 use File::Slurp qw( read_file );
 use Log::Any qw( $log );
 use Readonly;
+use Zonemaster::Backend::Validator qw( :untaint );
+
+Readonly my %DB_MODULE_NAMES => (
+    sqlite     => 'SQLite',
+    mysql      => 'MySQL',
+    postgresql => 'PostgreSQL',
```

Comment:
Actually yes! I tried to get rid of it but we do use it to derive the name of the database adapter module.

Reply:
A good reason! I suggest just a small comment, maybe right here in the code, why we have that.

Reply:
Have you considered a comment at `_normalize_engine_type` why the camel case forms are set here so that we do not have to think about it again? The one line that you states above would be enough.

Reply:
I think you'll find out soon enough if you try to change it. You'll get complaints about "Can't locate Zonemaster/Backend/DB/mysql.pm". I think that's quite clear.

Reply:
Why are you against adding a comment, documentation, in the code?

Reply:
Because in this particular case I think it adds more noise than value. More documentation isn't always better.

Reply:
I disagree that it would add to noise. It would clarify the choice. You are not the only one reading the code. With a comment I ask for this thread would never have been started.